http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -239,6 +239,16 @@ object clusterQuery extends TableQuery(new ClusterTable(_)) {
     fullClusterQuery(baseQuery)
   }
 
+  def fullClusterRecordQueryByUniqueKey(googleProject: GoogleProject,
```

Comment:
MInor: we usually name these methods `full` when they do all the joins… maybe name this `clusterRecordQueryByUniqueKey`?